commands/curation/curationaudit.go

```diff
@@ -53,6 +55,8 @@ const (
 
 	MinArtiPassThroughSupport = "7.82.0"
 	MinXrayPassTHroughSupport = "3.92.0"
+	MinArtiMavenSupport       = "7.82.0"
+	MinArtiXraySupport        = "3.92.0"
```

Comment:
```suggestion
	MinArtiMavenSupport       = "7.82.0"
	MinArtiXraySupport        = "3.92.0"
```
whay are those? delete if not used